README.md

```diff
@@ -68,8 +68,28 @@ To get your JWT token, visit <https://store.gatsbyjs.org> and log in with your G
 3.  Select `access_token` in the main panel
 4.  Copy the JWT token, which will be a value starting with `ey`
 
-### Step 3: Send a test request using cURL
+### Step 3: Configure `env` variables
+
+The `.env.EXAMPLE` file contains a list of `env` variables used in various locations throughout the repository. Configure each variable's value per your environment setup and credentials.
+
+| Name                    | Description                                                       |
+| ----------------------- | ----------------------------------------------------------------- |
+| `APP_LOGLEVEL`          | The debug log output level. Default: `info`                       |
+| `APP_HOST`              | The hostname of the API server. Default: `localhost`              |
+| `APP_PORT`              | The port number for the API server. Default: `8080`               |
```

Comment:
As far as I can tell, these don't have defaults, they're just pre-populated in the `.env.EXAMPLE`. (ie if you run the app without having a `.env`, these variables will be undefined)